llvm/test/tools/llvm-objcopy/tool-options.test

```diff
@@ -0,0 +1,4 @@
+## An error must be reported if a required argument value is missing.
+# RUN: not llvm-objcopy --only-section 2>&1 | FileCheck --check-prefix=CHECK-NO-VALUE %s
+# RUN: not llvm-objcopy -O 2>&1 | FileCheck --check-prefix=CHECK-NO-VALUE %s
+# CHECK-NO-VALUE: error: argument to '{{.*}}' is missing
```

Comment:
Why are you using a wildcard pattern here? It seems sensible to properly check the message generated. This also is only testing part of the message, but there are other parts that are important too.

Reply:
I am using a wildcard to reuse one pattern for several (two) different options. The point of this test is to verify that the message gets printed. I think there is enough context in the message to avoid misidentification for other messages.
I am not checking if the message is exactly the same because I don't see it as a realistic possibility. 
I don't think a random corruption in the output is likely, and testing for it would be outside of the scope of this test.

Reply:
> I think there is enough context in the message to avoid misidentification for other messages.

Agreed.

> I don't think a random corruption in the output is likely, and testing for it would be outside of the scope of this test.

This isn't why I think you should check the exact contents. Checking the exact contents explicitly will show that you've used the correct `%*` stuff in your `createStringError` call. Equally, should the code change in the future to use a different mechanism to parse the command-line/report the error/etc, it would show that the change is still reporting the context correctly.